armi/materials/uraniumOxide.py

```diff
@@ -162,7 +164,7 @@ def meltingPoint(self):
         """
         return self.__meltingPoint
 
-    def density(self, Tk: float = None, Tc: float = None) -> float:
+    def _density(self, Tk: float = None, Tc: float = None) -> float:
```

Comment:
Yeah, that's going to be a big push at some point soon.

BUT let's not force that separate ticket on Tommy for this PR.  He's just fixing a bug.

I think that other ticket will force big changes on the simulation results of downstream ARMI plugins and apps.  Which maybe is what we want. But it will definitely require a few of it's own PRs, so that we can _very carefully_ test HOW downstream modeling results change.

It's import, for sure. But we also have a separate ticket for that change.